src/components/transactions/HexEncodedData/index.tsx

```diff
@@ -28,7 +29,7 @@ export const HexEncodedData = ({ hexData, title, limit = 20 }: Props): ReactElem
         <>
           {showTxData ? hexData : shortenText(hexData, 25)}{' '}
           <Link component="button" onClick={toggleExpanded}>
```

Comment:
I believe you could also do `type=button` to prevent it from triggering a form

Reply:
It does indeed. I much prefer that. I've switched it over.